### PR TITLE
xcode integration + simple configuration file

### PR DESCRIPTION
**Pull request by dmueller39**

### CODE REVIEW

File: src/ocstyle/main.py [hunk 1]
```diff
@@ -48,13 +49,26 @@ def main():
...
+  
+  config = {}
+  
+  if os.path.isfile(".ocstyle.yml"):
+    with open(".ocstyle.yml", 'r') as stream:
+      config = yaml.safe_load(stream)
+  
+  for fpath in config["included"]:
+    for root, dirs, files in os.walk(fpath):
+      for file in files:
```
Comment (datadeflator) [inline src/ocstyle/main.py]: http://stackoverflow.com/questions/24942358/is-file-a-keyword-in-python

Technically, 3rd in tuple is filenames so perhaps rename to "filename" and for filename in filenames?

**Comment by datadeflator** [comment]
One comment otherwise looks good

**Comment by dmueller39** [comment]
Updated with amended commit

File: src/ocstyle/main.py [hunk 2]
```diff
@@ -47,14 +48,27 @@ def main():
...
+  for fpath in config["included"]:
+    for root, dirs, filenames in os.walk(fpath):
+      for filename in filenames:
+        if filename.endswith(".h") or filename.endswith(".m"):
+           source_filenames.append(os.path.join(root, filename))
+  
+  for source_filename in source_filenames:
+    absfilepath = os.path.abspath(source_filename)
+    if not os.path.isdir(source_filename):
       print filename
```
Comment (datadeflator) [inline src/ocstyle/main.py]: should be: print source_filename

right?

Comment (dmueller39) [inline src/ocstyle/main.py]: Yes!

**Comment by dmueller39** [comment]
Updated again

**Pull request merged**